CMake/ClangFormatHelper.cmake

```diff
@@ -0,0 +1,45 @@
+# =============================================================================
```

Comment:
I am very surprised to see this file. I would love to see hpc-coding-standard used here instead.
Is at least the .clang-format file the one specified in the BBP coding standard?

Reply:
Absolutely, @pramodk said he'll defer this to a seperate PR together with the formatting changes applied. Makes sense, I think.

Reply:
yeah agree, this should definitely go with use of hpc conding convention and I have already created issue https://github.com/BlueBrain/CoreNeuron/issues/200.

Note that this file new file is just extracted from [existing CMake](https://github.com/BlueBrain/CoreNeuron/blob/master/CMakeLists.txt#L231). I will work on this in a separate PR.